code/datums/gamemode/factions/bloodcult/bloodcult_runes.dm

```diff
@@ -475,6 +475,10 @@ var/list/rune_appearances_cache = list()
 			if(istype(V))
 				virus2["[V.uniqueID]-[V.subID]"] = V.getcopy()
 
+/obj/effect/rune/clean_act(var/cleanliness)
+	if ((cleanliness >= CLEANLINESS_SPACECLEANER) && (!activated))
```

Comment:
how do you clean activated cultrunes?
currently you need to use a cleanerspray, i remember using mop+water once and it didn't clean

Reply:
You uncurse them with holy salts (salt + holy water), if I'm not mistaken.

Reply:
Regular salt works too, it just takes longer.

Reply:
damn you're right, cleanerspray doesn't work on runes
surprised i haven't seen more hallway cults with how hard it is to erase the things